docs/python/testing.md

```diff
@@ -163,7 +46,13 @@ If the test discovery succeeds, you'll see tests listed in the Test Explorer:
 
 ![The VS Code Test Explorer for Python tests](images/testing/test-explorer.png)
 
-If discovery fails (for example, the test framework isn't installed or you have a syntax error in your test file), you'll see an error message displayed in the Test Explorer. You can check the **Python** output panel to see the entire error message (use the **View** > **Output** menu command to show the **Output** panel, then select **Python** from the dropdown on the right side).
+When triggering test discovery directly from the Test Explorer, you can also cancel an ongoing test discovery call. Use the Cancel button, which replaces the Refresh button during discovery.
+
+![Cancel test discovery button in the Test Explorer.](/images/testing/test-cancel-button.png)
+
+If discovery fails (for example, the test framework isn't installed or you have a syntax error in your test file), you'll see an error message displayed in the Test Explorer. The Python extension utilizes error tolerant discovery, meaning if there is an error during discovery that effects some tests but not all, you will see a node of tests and an error node in the Test Explorer. You can check the **Python** output panel to see the entire error message (use the **View** > **Output** menu command to show the **Output** panel, then select **Python** from the dropdown on the right side).
+
+> **Tip**: Unittests can be discovered by pytest, but not vice versa. In the case where testing is configured using unittest but tests are written in pytest, there will be an error message in the Test Explorer.
```

Comment:
there will not be an error message it will just not find the tests. So if you have pytests but you select unittests then we wont be able to find any tests in the workspace

Reply:
![image](https://github.com/user-attachments/assets/13e3c6fd-3332-4416-819b-a698afcd2385)
I see this in the Test Explorer....is this different or unexpected?